rj_gameplay/rj_gameplay/action/line_kick.py

```diff
@@ -0,0 +1,50 @@
+"""This module contains the interface and action for kick."""
+
+from abc import ABC, abstractmethod
+
+import stp.role as role
+import stp.action as action
+import stp.rc as rc
+import numpy as np
+from rj_msgs.msg import RobotIntent, MotionCommand, EmptyMotionCommand, LineKickMotionCommand
+from rj_geometry_msgs.msg import Point
+from typing import Optional
+import math
+from rj_msgs import msg
+
+class LineKickAction(action.IFiniteAction):
+    """
+    Activates kicker. Intended to be used after an aim/drive action in a skill.
+    """
+    def __init__(self, robot_id: int, target: np.ndarray, priority: int = 0) -> None: 
+        self.robot_id = robot_id
+        self.priority = priority
+        # should be ball's position
+        self.target = target
+
+        self.count = -1
+
+    def tick(self, intent: msg.RobotIntent) -> msg.RobotIntent:
+        line_kick_command = LineKickMotionCommand()
+        line_kick_command.target = Point(x=self.target[0],y=self.target[1])
+        # line_kick_command.target.x = self.target[0]
+        # line_kick_command.target.y = self.target[1]
+        """
+        # not sure if this is how you ref this constant (in RobotIntent.msg)
+        intent.shoot_mode = intent.SHOOT_MODE_KICK 
+
+        self.count += 1
+        if self.count == 0:
+            intent.kick_speed = 255.0 # kMaxKick is in rj_constants, not sure how to reference it
```

Comment:
Yeah, this is still a sticking point...want me to write a global param client for the Python side?

Reply:
That would be nice. @hdemusg and I tried to workaround this with [a constants file in stp/utils](https://github.com/RoboJackets/robocup-software/commit/d7dd036b3932e6f0399ddcf71e3ad7adb5c36a46), but I assume that's not the right way?